src/pages/extension-manager/extension-developed-by-adobe/content-fragments-workflows/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Content Fragments workflows
+description: Explore enhanced functionality in the Content Fragments Console and AEM Content Fragment Editor, simplifying workflow initiation.
+contributors:
+- AdobeDocs/uix
+---
+
+# Content Fragments workflows
+
+This extension provides UI extensibility, allowing customers to initiate an AEM workflow for selected content fragments.
+
+##  Extension overview
+
+The extension enhances two AEM services: [AEM Content Fragments Console](../../../services/aem-cf-console-admin/) and [AEM Content Fragment Editor](../../../services/aem-cf-editor/). It adds a button to the `Action Bar` for the Content Fragments Console and a button to the `Header Menu` for the AEM Content Fragment Editor.
```

Comment:
This sentence sounds a little strange to me. Would be nice to understand what exactly this button does from the beginning. I suggest moving an information from the last sentence in this paragraph somewhere higher.

Reply:
Moved the above sentence up.
`When clicked, it opens a modal dialog through which you can initiate a workflow.`